client/package.json

```diff
@@ -16,7 +16,7 @@
     "e2e-stage": "CXG_URL_BASE=https://cellxgene.staging.single-cell.czi.technology playwright test",
     "e2e-prod": "CXG_URL_BASE=https://cellxgene.cziscience.com playwright test",
     "fmt": "eslint --fix src __tests__",
-    "lint": "eslint src __tests__ & npm run type-check",
+    "lint": "eslint src __tests__ && npm run type-check",
```

Comment:
I think this with `&` only will run both commands in parallel. Is there a reason to make this sequential?